.github/workflows/kvrocks.yaml

```diff
@@ -35,7 +35,7 @@ concurrency:
 jobs:
   precondition:
     name: Precondition
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-22.04.4
```

Comment:
What's this? There is no `ubuntu-22.04.4`.

Please refer to https://github.com/actions/runner-images.

Reply:
😢 I thought we could choose a patch version that was built on 20230304, is it good to change ubuntu-22.04 to ubuntu-20.04?

Reply:
We can switch to 20.04 and wait for an update for 22.04.

Reply:
Sure, I revert the last commit.